dev/org.eclipse.codewind.core/build.properties

```diff
@@ -10,7 +10,7 @@
 ###############################################################################
 
 source.. = src/
-output.. = bin/
+output.. = binaries/
```

Comment:
Is this supposed to change?  I think output is the output dir for the Java classes.